core/services/job/job_orm_test.go

```diff
@@ -581,6 +582,53 @@ func TestORM_CreateJob_OCR_DuplicatedContractAddress(t *testing.T) {
 	})
 }
 
+func TestORM_CreateJob_OCR2_DuplicatedContractAddress(t *testing.T) {
+	customChainID := utils.NewBig(testutils.NewRandomEVMChainID())
+
+	config := configtest.NewGeneralConfig(t, func(c *chainlink.Config, s *chainlink.Secrets) {
+		enabled := true
+		c.EVM = append(c.EVM, &evmcfg.EVMConfig{
+			ChainID: customChainID,
+			Chain:   evmcfg.Defaults(customChainID),
+			Enabled: &enabled,
+			Nodes:   evmcfg.EVMNodes{{}},
+		})
+	})
+	db := pgtest.NewSqlxDB(t)
+	keyStore := cltest.NewKeyStore(t, db, config)
+	require.NoError(t, keyStore.OCR2().Add(cltest.DefaultOCR2Key))
+
+	lggr := logger.TestLogger(t)
+	pipelineORM := pipeline.NewORM(db, lggr, config)
+	bridgesORM := bridges.NewORM(db, lggr, config)
+
+	cc := evmtest.NewChainSet(t, evmtest.TestChainOpts{DB: db, GeneralConfig: config})
+	jobORM := NewTestORM(t, db, cc, pipelineORM, bridgesORM, keyStore, config)
+
+	require.NoError(t, evm.NewORM(db, lggr, config).EnsureChains([]utils.Big{*customChainID}))
+
+	// defaultChainID := config.DefaultChainID()
+
+	_, address := cltest.MustInsertRandomKey(t, keyStore.Eth())
+	//_, bridge := cltest.MustCreateBridge(t, db, cltest.BridgeOpts{}, config)
+	//_, bridge2 := cltest.MustCreateBridge(t, db, cltest.BridgeOpts{}, config)
+
+	jb, err := ocr2validate.ValidatedOracleSpecToml(config, testspecs.OCR2EVMSpecMinimal)
+	jb.Name = null.StringFrom("Job 1")
+	jb.OCR2OracleSpec.TransmitterID = null.StringFrom(address.String())
+	require.NoError(t, err)
+
+	err = jobORM.CreateJob(&jb)
+	require.NoError(t, err)
+
+	jb2, err := ocr2validate.ValidatedOracleSpecToml(config, testspecs.OCR2EVMSpecMinimal)
```

Comment:
Oh, this error is not checked. It looks like the previous validate drifted a few lines away too.

Reply:
This one is still unchecked:
```suggestion
	jb2, err := ocr2validate.ValidatedOracleSpecToml(config, testspecs.OCR2EVMSpecMinimal)
	require.NoError(t, err)
```